extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -134,7 +134,7 @@ function PanelStudyBrowserTracking({
       const actuallyMappedStudies = mappedStudies.map(qidoStudy => {
         return {
           studyInstanceUid: qidoStudy.StudyInstanceUID,
-          date: formatDate(qidoStudy.StudyDate),
+          date: formatDate(qidoStudy.StudyDate) || 'No Study Date',
```

Comment:
This is what was suggested in the reporting issue. However we can discuss alternatives.

Reply:
How about N/A?

Reply:
N/A could apply to anything... not just a date. If someone sees N/A, they do not know what it is referring to. So no, I do not think N/A is appropriate. How about just 'No Date'.

Reply:
Also N/A stands for not applicable. So I really don't think that is right.

Reply:
@wayfarer3130 What should be here if it is empty?

Reply:
For reference, notice in the video below, the viewport action bar does not wrap when 'No Study Date' is present.


https://github.com/OHIF/Viewers/assets/109477394/489dc352-c375-4704-b507-1a16a7322581

Reply:
Probably it should be an empty string if it is missing, then the UI can decide on the value to show when empty.

Reply:
@wayfarer3130, this is the UI. For me, the question is what to show in the UI.

Reply:
What about 'No Study Data' or 'No Date'

Reply:
I'm asking Dan about what to show.

Reply:
Agreed with No Study Date